Maple2.Server.Game/PacketHandlers/TutorialItemHandler.cs

```diff
@@ -0,0 +1,39 @@
+using System.Linq;
+using Maple2.Model;
+using Maple2.Model.Game;
+using Maple2.Model.Metadata;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.PacketHandlers;
+using Maple2.Server.Game.Session;
+
+namespace Maple2.Server.Game.PacketHandlers;
+
+public class TutorialItemHandler : PacketHandler<GameSession> {
+    public override RecvOp OpCode => RecvOp.RequestTutorialItem;
+
+    public override void Handle(GameSession session, IByteReader packet) {
+        var jobTable = session.TableMetadata.JobTable;
+
+        var jobCode = session.Player.Value.Character.Job.Code();
+
+        if (!jobTable.Entries.TryGetValue(jobCode, out var jobMetadata)) return;
+        
+        if (session.Field != null && (session.Player.Value.Character.Level > 1 || jobMetadata.Tutorial.StartField != session.Field.MapId)) return;
```

Comment:
nit: I would check `Level` up top to return as early as possible. the `StartField` check can remain here since we need `jobMetadata` first.